lib/order.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :basket, :menu
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @basket = []
+    @total = 0
+    @confirmed = false
+    time = (Time.now+(60*60))
+    @confirmation_text = "Thank you! Your order was placed and will be delivered before #{time.strftime("%H")}:#{time.strftime("%M")}. You ordered:"
+  end
+
+  def add_item(item, quantity = 1)
+    check_exists(item)
+    @confirmation_text += " #{quantity} portion(s) of #{item},"
+    retrieve_price(item)
```

Comment:
This and the following line are unnecessary.  Just calculate the total on the fly when it is requested.